tests/src/test/scala/whisk/core/entity/test/ExecManifestTests.scala

```diff
@@ -170,4 +176,22 @@ class ExecManifestTests
                     image.localImageName("r", "p", Some("tag")) shouldBe s"r/p/$name:tag"
             }
     }
+
+    it should "throw an error when configured manifest is invalid" in {
+        val config_manifest = "{\"nodejs\":[{\"kind\":\"nodejs\",\"image\":{\"name\":\"nodejsaction\"},\"deprecated\":true},{\"kind\":\"nodejs:6\",\"default\":true,\"image\":{\"name\":\"nodejs6action\"}}]}"
+        val file = File.createTempFile("cxt", ".txt")
+        file.deleteOnExit()
+
+        val bw = new BufferedWriter(new FileWriter(file))
+        bw.write("runtimes.manifest=" + config_manifest + "\n")
+        bw.close()
+
+        var result:Try[_] = ExecManifest.initialize(new WhiskConfig(Map("runtimes.manifest" -> null), Set(), file), true)
```

Comment:
Good catch. I could change this to prevent consul, but at the same time I have another PR in the works to remove consul from WhiskConfig. In this light should we worry about this in the test ?